### PR TITLE
Modified: `IcmpHandler` doesn't send packets to `Encoder` but sends t…

### PR DESCRIPTION
**Pull request by agafox**
…hem back to `Ipv4Handler`

### CODE REVIEW

File: src/main/kotlin/io/sip3/captain/ce/pipeline/IcmpHandler.kt [hunk 1]
```diff
@@ -17,48 +17,27 @@
...
-    }
+class IcmpHandler(val ipv4Handler: Ipv4Handler, vertx: Vertx, bulkOperationsEnabled: Boolean) : Handler(vertx, bulkOperationsEnabled) {
 
     override fun onPacket(buffer: ByteBuf, packet: Packet) {
-        val offset = buffer.readerIndex()
-
-        val type = buffer.getByte(offset).toInt()
-        val code = buffer.getByte(offset + 1).toInt()
+        // Type
+        val type = buffer.readByte().toInt()
```
Comment (sip3leo) [inline src/main/kotlin/io/sip3/captain/ce/pipeline/IcmpHandler.kt]: Is it really necessary to convert `type` and `code` to Integer?

File: src/main/kotlin/io/sip3/captain/ce/pipeline/RouterHandler.kt [hunk 2]
```diff
@@ -50,18 +50,26 @@ class RouterHandler(vertx: Vertx, bulkOperationsEnabled: Boolean) : Handler(vert
             // RTP or RTCP packet
             val packetType = buffer.getUnsignedByte(offset + 1).toInt()
             if (packetType in 200..211) {
-                rtcpHandler.handle(buffer, packet)
+                // Doesn't matter if RTCP packet was rejected or not
```
Comment (sip3leo) [inline src/main/kotlin/io/sip3/captain/ce/pipeline/RouterHandler.kt]: I think that if comment is correct, we don't need to check `packet.rejected` below in code.

Comment (agafox) [reply src/main/kotlin/io/sip3/captain/ce/pipeline/RouterHandler.kt]: Modified comment message

File: src/main/kotlin/io/sip3/captain/ce/pipeline/IcmpHandler.kt (hunk 1 above)
Comment (agafox) [reply src/main/kotlin/io/sip3/captain/ce/pipeline/IcmpHandler.kt]: Yes, because we use them in `if` statement and operator `==` can't be applied to Byte

**Pull request merged**